src/HostedExplorerChildFrame.ts

```diff
@@ -6,13 +6,17 @@ export interface HostedExplorerChildFrame extends Window {
   hostedConfig: HostedConfig;
 }
 
-export interface AAD {
+interface AADTokenAuth {
+  aadToken: string;
+}
```

Comment:
Move this down to the `AAD` interface below. None of the other auth types will have an AAD token.